calico/datamodel_v1.py

```diff
@@ -85,6 +86,10 @@
 ENDPOINT_STATUS_DOWN = "down"
 ENDPOINT_STATUS_ERROR = "error"
 
+# Information intended for use by the DHCP agent.
+DHCP_DIR = ROOT_DIR + "/dhcp" + DHCP_VERSION
+SUBNET_DIR= DHCP_DIR + "/subnet"
```

Comment:
I'm thinking that we may, later, want to have some config attached to the whole OpenStack-style network and this prevents that